### PR TITLE
[FIX] Flex basis for correct width on SingleLineInputBase

### PR DESCRIPTION
**Pull request by stefaandevylder**
### Description

- Fix the flex basis in SingleLineInputBase so the datepicker is correctly positioned when using a width of 100%.

#### Screenshot before this PR

Screenshot 2022-02-28 at 10 39 59

#### Screenshot after this PR

Screenshot 2022-02-28 at 10 41 15
^ Look at the two input fields, they are now the same size. In the screenshot before they are 1px off.

### Breaking changes

N/A

### CODE REVIEW

**Review by qubis741 — DISMISSED** [review]

**Review by JorenSaeyTL — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 ### Fixed
 
+- `SingleLineInputBase`: Fixed `flex` basis setting. ([@stefaandevylder]https://github.com/stefaandevylder) in [#2014](https://github.com/teamleadercrm/ui/pull/2014)
```
Comment (lowiebenoot) [inline CHANGELOG.md:13]: This is not really an explanation of what you're fixing. I assume you're fixing that the datepicker wasn't full width?

Comment (stefaandevylder) [reply CHANGELOG.md:13]: Yeah but this applies for more than the datepicker I thought, or am I wrong?

Comment (JorenSaeyTL) [reply CHANGELOG.md:13]: Basically when width was set to 100% the input width was 2px off. with for example setting the select input to 100%. The select however was styled correctly, but the SingleLineInputBase 2px too big. Also missing a '(' after your name. I'll fix it

Comment (JorenSaeyTL) [reply CHANGELOG.md]: Updated the section and fixed brackets

**Review by JorenSaeyTL — APPROVED** [review]

**Review by lowiebenoot — APPROVED** [review]

**Comment by JorenSaeyTL** [comment]
MCed ✅

**Pull request merged**